modules/ROOT/edits/enable-inbound-traffic-self.adoc

```diff
@@ -0,0 +1,22 @@
+= Configure Domains and Inbound Traffic (On Self-Managed Kuberenetes)
+
+To enable external applications to connect to Runtime Fabric on Self-Managed Kubernetes, you must provide an ingress controller that is compatible with your Kubernetes environment. Runtime Fabric uses the ingress controller to route requests to deployed applications.
+
+Inbound traffic is enabled by default. This allows Mule applications and API gateways to receive inbound connections. You can disable inbound traffic per application in Runtime Manager when deploying or configuring an application.
+
+Runtime Fabric allows you to configure domains that serve as endpoints for your deployed applications.
+
+== Configure Domains for Runtime Fabric on Self-Managed Kubernetes
+
+. Navigate to Runtime Manager and select *Runtime Fabric*.
+. Select the name of your Runtime Fabric to open the management page.
+. Select *Inbound Traffic*, then click *Add Endpoints.*
+. Add the endpoint provided by your IT administrator, including the `http://` or `https://` protocols. Separate multiple endpoints with commas.
+. Confirm your changes by clicking the *Add* button.
```

Comment:
Do you mean "commit" rather than "confirm" (which to me implies verifying or checking that something has been done, not actually doing it)?